### PR TITLE
Create direct message interface

### LINKED ISSUE #108: Create direct message interface
### What?
Adds a direct provider interface to send a message based on a channel id.

### PR DESCRIPTION
**Pull request by scopsy**
Adds the ability to create direct providers that allow sending a message to a specific `$channel_id`. In a later point, we can add support for private messages and thread replies.

Closes #108

### CODE REVIEW

**Comment by davidsoderberg** [comment]
Should direct messages have attachments?

**Comment by scopsy** [comment]
@davidsoderberg I think for now we can do it without, and later we can map all providers of attachments support and release it later if needed

File: packages/core/src/lib/handler/direct.handler.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    let content = '';
+    if (typeof this.message.template === 'string') {
+      content = compileTemplate(this.message.template, data);
+    } else {
+      content = await this.message.template(data);
+    }
+
+    if (!data.$channel_id) {
+      throw new Error(
+        '$channel_id is missing in trigger payload. To send an SMS You must specify a $phone property.'
```
Comment (ComBarnea) [inline packages/core/src/lib/handler/direct.handler.ts:18]: @scopsy this needs to be updated fot direct provider error

**Review by ComBarnea — CHANGES_REQUESTED** [review]
See changes requested in comments

**Pull request merged**